Makefile

```diff
@@ -8,6 +8,18 @@ else
 	GOOS=linux
 endif
 
+# The latest git tag on branch
+GIT_TAG := $(shell git describe --abbrev=0 --tags)
+# If the latest commit is tagged
+TAGGED_COMMIT := $(shell git tag -l --contains HEAD)
+REVISION := $(shell git rev-parse --short HEAD)
```

Comment:
should this be `REVISION ?= ...` so that a user's env can supply `REVISION`, if set?